wp-content/themes/currentorg/less/wpbdp.less

```diff
@@ -300,11 +300,11 @@
       line-height: 2 * @common-whitespace;
       padding: 0 0 0 @common-whitespace;
       box-sizing: border-box;
+      border-radius: 0;
 
-      border-top: 1px solid @grey-l;
-      background: none;
+      background: #e86059;
```

Comment:
This should be the LESS variable `@highlight` in case we need to get a different color here in the future.

Reply:
Good catch. Updated.